src/Tgstation.Server.Host/ServerFactory.cs

```diff
@@ -49,7 +49,11 @@ public async Task<IServer> CreateServer(string[] args, string updatePath, Cancel
 
 			var basePath = IOManager.ResolvePath();
 			IHostBuilder CreateDefaultBuilder() => Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder(args)
-				.ConfigureAppConfiguration((context, configuration) => configuration.SetBasePath(basePath));
```

Comment:
Unsure if removing `configuration.SetBasePath()` here is gonna break stuff. It didnt in my testing but who knows.